public/angular/index.html

```diff
@@ -43,6 +43,7 @@
         <script src="/js/Autolinker.min.js"></script>
         <script src="/js/canvasjs.min.js"></script>
         <script src="/js/app.min.js"></script>
+        <script src="https://d3js.org/d3.v6.min.js"></script>
```

Comment:
A more conventional place to put this would be in package.json via `npm install --save d3` then import `d3` in files explicitly rather than using a global constant.

Reply:
Yeah, I think I tried that initially, and it didn't work for whatever reason, so I just said screw, we'll go with the CDN for the sake of getting something that works. But I'll try it again.

Reply:
OK, yeah, the reason was that the d3 library doesn't play well with our build system. I guess it uses newer syntax or something, but you'll get this error if you don't use the CDN:
```
[11:44:04] Using gulpfile ~/Documents/web/aggie/gulpfile.js
[11:44:04] Starting 'build'...
[11:44:04] Starting 'debugTranslations'...
[11:44:04] Starting 'angular'...
[11:44:04] Finished 'debugTranslations' after 65 ms
[11:44:07] Plumber found unhandled error:
 Error: StringMap expected string key
[11:44:07] Finished 'angular' after 2.98 s
[11:44:07] Finished 'build' after 2.98 s```

Reply:
I think you're right, seems like modern versions of D3 don't play well with the angular transpiler we're using (https://stackoverflow.com/a/45980599/763231 for example). This is fine for now then, thanks for the context.